NorthstarDedicatedTest/rpakfilesystem.cpp

```diff
@@ -119,6 +119,10 @@ void LoadCustomMapPaks(char** pakName, bool* bNeedToFreePakName)
 	}
 }
 
+#include "pch.h"
+// these are hashed with STR_HASH
+std::unordered_map<int32_t, size_t> loadedPaks {};
```

Comment:
would be better to use some kind of bidirectional map here but i dont think standard libs have one, could use 2 maps?
also there's no reason this shouldn't be in the PakLoadManager